native/python3/Makefile

```diff
@@ -31,7 +31,7 @@ PIP_NATIVE = $(WORK_DIR)/../../../native/$(PKG_NAME)/work-native/install/usr/loc
 python3_native_post_install: $(WORK_DIR)/python-native.mk
 	@$(MSG) Installing pip, setuptools, cffi and cross env
 	@$(RUN) wget https://bootstrap.pypa.io/get-pip.py
-	@$(RUN) $(PYTHON) get-pip.py "pip<20.3"
+	@$(RUN) $(PYTHON) get-pip.py "pip==21.1.1"
```

Comment:
Is this expected ? I would have thought you'd only need to change python 3.8 related resources.
The usage of a specific version is a good idea anyway.

Reply:
python 3.7 changes moved to #4637